client/dashboard/sites/hosting-feature/upsell.tsx

```diff
@@ -72,9 +60,13 @@ export default function HostingFeatureUpsell( {
 				</>
 			}
 			actions={
-				<Button variant="primary" size="compact" onClick={ handleUpgradePlan }>
-					{ __( 'Upgrade plan' ) }
-				</Button>
+				<UpsellCTAButton
+					size="compact"
+					text={ __( 'Upgrade plan' ) }
+					trackId={ tracksFeatureId }
```

Comment:
Super nit: the product name is `Tracks` (with `s`); could we rename the prop (e.g. `tracksId`) here and in `OverviewCard` 🙈